WalletWasabi/Wallets/FileSystemBlockRepository.cs

```diff
@@ -46,6 +154,11 @@ public async Task<Block> GetAsync(uint256 hash, CancellationToken cancellationTo
 					{
 						var blockBytes = await File.ReadAllBytesAsync(filePath, cancellationToken).ConfigureAwait(false);
 						block = Block.Load(blockBytes, Network);
+
+						new FileInfo(filePath)
```

Comment:
if we remove the prune we don't need this.

Reply:
We don't.